File: SDKs/ote-godaddy.com/certificates/1.0.0/typescript/src/sdk/models/operations/certificatecancel.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class CertificateCancelPathParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=certificateId" })
  certificateId: string;
}


export class CertificateCancelRequest extends SpeakeasyBase {
  @SpeakeasyMetadata()
  pathParams: CertificateCancelPathParams;
}


export class CertificateCancelResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  error?: any;

  @SpeakeasyMetadata()
  statusCode: number;
}

File: SDKs/ote-godaddy.com/certificates/1.0.0/python/src/sdk/models/operations/certificate_cancel.py
import dataclasses
from typing import Any,Optional


@dataclasses.dataclass
class CertificateCancelPathParams:
    certificate_id: str = dataclasses.field(metadata={'path_param': { 'field_name': 'certificateId', 'style': 'simple', 'explode': False }})
    

@dataclasses.dataclass
class CertificateCancelRequest:
    path_params: CertificateCancelPathParams = dataclasses.field()
    

@dataclasses.dataclass
class CertificateCancelResponse:
    content_type: str = dataclasses.field()
    status_code: int = dataclasses.field()
    error: Optional[Any] = dataclasses.field(default=None)
    
